pkg/jiraservicedesk/client/client_project.go

```diff
@@ -147,13 +155,43 @@ func (c *jiraServiceDeskClient) DeleteProject(id string) error {
 		return err
 	}
 
-	if response.StatusCode != 204 {
+	if response.StatusCode < 200 || response.StatusCode > 299 {
 		return errors.New("Rest request to delete Project failed with status: " + strconv.Itoa(response.StatusCode))
 	}
 
 	return err
 }
 
+func (c *jiraServiceDeskClient) UpdateCustomerAccessStatus(key string) error {
+	body := CustomerAccessRequestBody{
+		autocompleteEnabled:     false,
+		manageEnabled:           false,
+		serviceDeskOpenAccess:   false,
+		serviceDeskPublicSignup: false,
+	}
```

Comment:
Why are these values hard-coded here ? As discussed we were supposed to add a new key in `customers` CRD